huntsman/utils/config.py

```diff
@@ -110,15 +113,14 @@ def start_config_server(host=None, port=6563, name='config_server',
         #Register with pyro & the name server
         uri = daemon.register(config_server)
```

Comment:
Consider using a more obvious variable than `uri` unless this is standard in pyro.